sdk/communication/azure-communication-administration/src/main/java/com/azure/communication/administration/CommunicationIdentityClientBuilder.java

```diff
@@ -70,12 +71,29 @@ public CommunicationIdentityClientBuilder pipeline(HttpPipeline pipeline) {
     /**
      * Set credential to use
      *
-     * @param credential credential for signing, overridden by the
-     * pipeline field.
+     * @param accessKey access key for initalizing CommunicationClientCredential
      * @return CommunicationIdentityClientBuilder
      */
-    public CommunicationIdentityClientBuilder credential(CommunicationClientCredential credential) {
-        this.credential = Objects.requireNonNull(credential, "'credential' cannot be null.");
+    public CommunicationIdentityClientBuilder accessKey(String accessKey) {
+        Objects.requireNonNull(accessKey, "'accessKey' cannot be null.");
+        this.credential = new CommunicationClientCredential(accessKey);
```

Comment:
If the user doesn't have to create the `CommunicationClientCredential` instance directly, can that type be moved to implementation package?

Reply:
@srnagar Same as the ConnectionString comment, the `CommunicationClientCredential` is used in other packages as well. Let me know what your thoughts are on this.